TerminologService/valueSetToRoots.py

```diff
@@ -22,7 +23,23 @@ def expand_value_set(url: str, onto_server: str = TERMINOLOGY_SERVER_ADDRESS):
     if '|' in url:
         url = url.replace('|', '&version=')
     term_codes = SortedSet()
-    response = requests.get(onto_server + f"ValueSet/$expand?url={url}", cert=(SERVER_CERTIFICATE, PRIVATE_KEY))
+    version = MAPPING_ONTO_VERSION.get('canonical_address')
```

Comment:
Why use a file if you always get the same version?

Reply:
It ensures that the system remains adaptable and scalable as more version-specific requirements emerge.